app/views/friend_requests/received_request.html.erb

```diff
@@ -5,28 +5,35 @@
 
     <div class="col-8">
       <div class="card">
-        <% if current_user.pending_requests_from_users.count.positive? %>
+        <% if current_user.incoming_pending_friends.count.positive? %>
```

Comment:
**[REQUIRED]**
* Let's separate code logic from the views([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright). 
We could have this method in `app/helpers/friend_requests_helper.rb` 
```
module FriendRequestsHelper
  def incoming_pending_friends?
    current_user.incoming_pending_friends.count.positive?
  end
end
```
and then just call it from the views.
`<% if incoming_pending_friends? %>`
Methods defined in the `app/helpers` are automatically available in the views.

**More Resources**
* [Where Do I Put My Code?](http://codefol.io/posts/Where-Do-I-Put-My-Code)
* [The Beginner's Guide to Rails Helpers](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers)